services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -465,6 +472,8 @@ private void removeTopics(Set<String> topicsToRemove) {
     for (String topic: topicsToRemove) {
       customizedViewRepository.unSubscribeRoutingDataChange(topic, this);
       storeVersionRateLimiters.remove(topic);
+      stats.getStoreStats(Version.parseStoreFromKafkaTopicName(topic))
```

Comment:
Can we add a check for existing of the storeStats, otherwise it will unnecessarily add a start just to be removed later.

Reply:
Sure